.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @Sage-Bionetworks/sagebio-it  @Sage-Bionetworks/sage-image-librarians
```

Comment:
These groups still need to be given write access to the repo, but that can be done after merging